src/theory/strings/inference_manager.cpp

```diff
@@ -30,16 +28,16 @@ namespace CVC4 {
 namespace theory {
 namespace strings {
 
-InferenceManager::InferenceManager(TheoryStrings& p,
-                                   context::Context* c,
+InferenceManager::InferenceManager(context::Context* c,
                                    context::UserContext* u,
                                    SolverState& s,
                                    TermRegistry& tr,
+                                   ExtTheory* e,
```

Comment:
Why is this a pointer instead of a reference like the other arguments?

Reply:
Had followed another part of the code, this is now updated to be consistent everywhere.